src/ai/backend/client/cli/manager.py

```diff
@@ -204,3 +205,51 @@ def exclude_agents(agent_ids):
     except Exception as e:
         print_error(e)
         sys.exit(1)
+
+
+@main.command()
+@click.argument('scaling_group', metavar='SCALING_GROUP', default='default')
+@click.argument('group', metavar='GROUP', default='default')
+@click.option('-a', '--all', is_flag=True,
+              help='Get all resources of group.')
+def get_resources(scaling_group, group, all):
+    '''
+    Get available resources from the scaling groups.
+    '''
+    config = get_config()
+    if config.endpoint_type != 'session':
+        print_warn('To use get-resources, your endpoint type must be "session".')
+        raise click.Abort()
+
+    with Session() as session:
+        ret = session.Resource.get_available_resources(scaling_group, group)
+        print(f'Total remaining resources of scaling group [{scaling_group}]:')
+        print('  CPU:', ret['scaling_group_remaining']['cpu'])
+        print('  Memory:', ret['scaling_group_remaining']['mem'])
```

Comment:
There may be more resource slot types.
Please use `session.Resource.get_resource_slots()` to retrieve the exact list of the currently configured resource slots, and then iterate over it when displaying the remainings.
Note that you may need to have an extra mapping from the resource slot name to a human-readable name with a fallback to the raw slot key.